.github/workflows/release.yml

```diff
@@ -58,3 +58,16 @@ jobs:
         if: ${{ steps.release.outputs.release_created }}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  homebrew:
+    name: "Bump Homebrew formula"
+    runs-on: ubuntu-latest
+    needs: release
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@b3327118b2153c82da63fd9cbf58942146ee99f0 # v3
+        with:
+          # A PR will be sent to github.com/Homebrew/homebrew-core to update this formula
+          formula-name: atmos
```

Comment:
should be the formule name not action-docs?

Reply:
Good catch! 

Addressed https://github.com/npalm/action-docs/pull/735/commits/4c64767fca1aabbedb70383e5bf5e2279ce0e721